windows.md

```diff
@@ -312,6 +312,26 @@ Before you start, please make sure that you're running Windows 10 or Windows 11.
     It should print out a welcome message like this:<br><br>
     <img src="macos-6-docker-hello-world.png"><br><br>
 
+    If after running `docker run hello-world` command, you receive an error like this:
+
+    ```bash
+    Unable to write to the database. Exit code: 1
+      at Docker.Backend.Database.WriteKeys(ICollection\'1 keyValues) at Docker.Backend.Database.ModifyDatabase(ICollection`1 keyValuesToWrite, ICollection`1 keysToRemove)
+      at Docker.Backend.Database.Write(Settings settings)
+      at Docker.Backend.ContainerEngine.Linux.DoStart(Settings settings)
+      at Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
+      at Docker.Core.Pipe.NamedPipeServer.<>c**DisplayClass8_0.<Register>b**0(Object[] parameters)
+      at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
+    ```
+
+    1. Press <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>Delete</kbd> and click on Task Manager
+       <img src="windows-7-task-manager.webp"><br><br>
+
+    2. Click on the Performance tab and make sure your CPU has the virtualisation setting enabled
+       <img src="windows-8-virtualisation-enabled.png"><br><br>
+
+    3. In the case virtualization is not enabled, make sure your machine meets the [docker requirements](https://docs.docker.com/desktop/windows/troubleshoot/#virtualization) and follow the steps in the Microsoft Windows [official documentation page](https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1)
```

Comment:
capitaliztion of `Docker`

Reply:
https://github.com/upleveled/system-setup/pull/16/commits/8157cbb674694df4b5a2d0a9512cbdb6af4c564c